packages/storybook/src/nimble/select/select.stories.ts

```diff
@@ -28,20 +28,22 @@ interface SelectArgs {
     optionsType: ExampleOptionsType;
```

Comment:
I just noticed we don't actually document the `value` property on the select. I think we should, since we mention it in the initial summary and in the docs for the options. You can use the category `nonAttributeProperties` (not sure if that's the exact name) to indicate it's a property, not an attribute.

I still have a task in #824 to document a recommendation to use form association rather than events/value, but we should still document the existence of `value`.

Reply:
When you do this, you could consider removing the text about `value` from the summary at the top; it's probably more detail than is needed for someone who is just trying to understand what the control does.